src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -345,13 +345,15 @@ These tasks can be easily integrated to your AI pipelines/workflows as you can s
 
 The management of the life cycle of ML MaaS will be detailed in the next sub-sections.
 
-Once an ML MaaS instance is up and running, it could be used for:
+Once a ML MaaS instance is up and running, it could be used for:
 
-- *AI Model Deployment or Update*: the user has to provide a valid identifier in order to deploy the model of his/her choice.
+- *AI Model Deployment or Update*: the user has to provide a valid specific AI Model identifier in order to deploy the model of his/her choice.
 
-- *Call of Predictions*: the user requests to give predictions for a specific payload that has to be converted into json data in order to get prediction's values.
+- *Call of Predictions*: when a specific AI model is running, the user can request predictions for a specific payload. This later has to be converted into json data in order to get prediction's values.
```

Comment:
This lat*t*er

Reply:
Sure.

Reply:
prediction's values => prediction values